projects/1.16.1/assets/playerex/playerex/lang/zh_cn.json

```
@@ -0,0 +1,108 @@
+	"attribute.name.playerex.melee_crit_damage": "暴击伤害(近)",
+	"attribute.name.playerex.melee_crit_chance": "暴击机率(近)",
+	"attribute.name.playerex.evasion": "闪避",
+	"attribute.name.playerex.ranged_damage": "远程伤害",
+	"attribute.name.playerex.ranged_crit_damage": "暴击伤害(远)",
+	"attribute.name.playerex.ranged_crit_chance": "暴击机率(远)",
+	"attribute.name.playerex.lifesteal": "吸血",
+	"playerex.attribute.level": "等级: +%s (%s%s)",
+	"playerex.attribute.level.alt": "玩家等级,与附魔等级是分开的,每升级一级获得一点属性点.",
+	"playerex.attribute.skillpoints": "属性点: +%s",
+	"playerex.attribute.skillpoints.alt": "显示你还有多少属性点,用给增加下面那五个基础属性.",
+	"playerex.attribute.constitution": "体质: +%s",
+	"playerex.attribute.constitution.alt0": "+1 血量",
+	"playerex.attribute.constitution.alt1": "+1%% 击退抗性(递减)",
+	"playerex.attribute.strength": "力量: +%s",
+	"playerex.attribute.strength.alt0": "+0.02/秒 生命恢复",
+	"playerex.attribute.strength.alt1": "+0.25 近站伤害",
+	"playerex.attribute.strength.alt2": "+0.25 护甲(递减)",
+	"playerex.attribute.dexterity": "敏捷: +%s",
+	"playerex.attribute.dexterity.alt0": "+0.08 格/秒 移动速度(递减)",
+	"playerex.attribute.dexterity.alt1": "+5%% 暴击伤害(近)(递减)",
+	"playerex.attribute.dexterity.alt2": "+0.1 攻击速度(近)(递减)",
+	"playerex.attribute.dexterity.alt3": "+0.25 暴击伤害(远)",
+	"playerex.attribute.intelligence": "智力: +%s",
+	"playerex.attribute.intelligence.alt0": "+5%% 暴击伤害(远)(递减)",
+	"playerex.attribute.intelligence.alt1": "+2%% 吸血(递减)",
+	"playerex.attribute.intelligence.alt2": "+1%% 生命恢复与倍率(递减)",
+	"playerex.attribute.luckiness": "幸运: +%s",
+	"playerex.attribute.luckiness.alt0": "+1 运气.",
+	"playerex.attribute.luckiness.alt1": "+2%% 暴击机率(近)(递减)",
+	"playerex.attribute.luckiness.alt2": "+2%% 暴击机率(远)(递减)",
+	"playerex.attribute.luckiness.alt3": "+2%% 闪避(递减)",
+	"playerex.attribute.health": "血量: %s/%s",
+	"playerex.attribute.health.alt0": "当前生命 / 最大生命. \n生命恢复: 每秒恢复的生命. \n生命恢复倍率: 玩家每次恢复的额外增加的治疗量.",
+	"playerex.attribute.health.alt1": "生命恢复: +",
+	"playerex.attribute.health.alt2": "生命恢复倍率: +",
+	"playerex.attribute.armor": "护甲: +%s",
+	"playerex.attribute.armor.alt0": "护甲: 原版的盔甲属性. \n韧性: 原版的盔甲韧性属性.",
+	"playerex.attribute.armor.alt1": "韧性: +",
+	"playerex.attribute.attack_speed": "攻击速度: +%s",
+	"playerex.attribute.attack_speed.alt": "原版武器上的攻击速度.",
+	"playerex.attribute.melee": "近站伤害: +%s",
+	"playerex.attribute.melee.alt0": "暴击伤害: ",
+	"playerex.attribute.melee.alt1": "暴击机率: ",
+	"playerex.attribute.ranged": "远程伤害: +%s",
+	"playerex.attribute.ranged.alt0": "暴击伤害: ",
+	"playerex.attribute.ranged.alt1": "暴击机率: ",
+	"playerex.attribute.evasion": "闪避: +%s%s",
+	"playerex.attribute.evasion.alt": "完全躲避弹射物的概率.",
+	"playerex.attribute.lifesteal": "吸血: +%s%s",
+	"playerex.attribute.lifesteal.alt": "近战攻击恢复生命的比例.",
+	"playerex.attribute.luck": "运气: +%s",
+	"playerex.attribute.luck.alt": "原版幸运属性.",
+	"playerex.attribute.gravity": "重力: +%s",
+	"playerex.attribute.gravity.alt": "重力: 数值越小,跳得越高.",
+	"playerex.attribute.reach_distance": "手臂长度: +%s",
+	"playerex.attribute.reach_distance.alt": "你最远可以够到的距离",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
	"playerex.attribute.reach_distance.alt": "你的最远触及距离。",
```